client/src/components/navbar-component/Navbar.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import logo from "../../../public/img/tces-logo.png";
+import Dropdown from "./Dropdown";
+import NavbarProfile from "./NavbarProfile";
+import NavbarButton from "./NavbarButton";
+
+function Navbar() {
+  const [isDropdownVisible, setIsDropdownVisible] = useState(false);
+
+  const toggleDropdown = () => {
+    setIsDropdownVisible(!isDropdownVisible);
+  };
+
+  return (
+    <div className="nav-container">
+      <div className="left-content">
+        <div className="image">
+          <img src={logo} alt="logo" width="46" height="50" />
```

Comment:
replace {logo} with "./img/tcesLogo.svg". svgs scale much better and are generally favourable over their png counterparts. Also please delete the tces-logo.png file from public/img